src/EnergyPlus/FluidCoolers.cc

```diff
@@ -2002,7 +2002,7 @@ void FluidCoolerspecs::update(EnergyPlusData &state)
 
     // Check if OutletWaterTemp is below the minimum condenser loop temp and warn user
     Real64 LoopMinTemp = state.dataPlnt->PlantLoop(this->plantLoc.loopNum).MinTemp;
-    if (this->OutletWaterTemp < LoopMinTemp && this->WaterMassFlowRate > 0.0) {
+    if (this->OutletWaterTemp<LoopMinTemp &&this->WaterMassFlowRate> 0.0) {
```

Comment:
Clang format comment: Reverse the order or create an intermediate variable

Reply:
Or just put the code back where it was and don't let VS reformat after that.

Reply:
Thanks @mjwitte and @Myoldmopar . The clang formatting worked ok after putting brackets to the two conditions--not bad given the little bit clarity added with the cost of a little bit redundancy.